src/Core/src/AbstractApi.php

```diff
@@ -76,16 +77,18 @@ public function __construct($configuration = [], ?CredentialProvider $credential
         ]));
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I always wondered what's the purpose of this. I always thougth that IDE will fallback to parent docblock.